submissions/kt-std/tiny-js-world/index.js

```diff
@@ -0,0 +1,57 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+   
+   function Inhabitant (species, name, gender, hands, legs, sayings, friends){
+   		this.species = species;
+   		this.name = name;
+   		this.gender = gender;
+   		this.hands = hands === 0 ? `${this.species}s don't have hands! They have paws`: hands;
+   		this.legs = legs;
+   		this.friends = Array.isArray(friends) && friends.length ? friends : 'Forever alone...';
+   		this.saySomething = sayings;
+   }
```

Comment:
It is good to see a constructor function as an OOP-oriented approach.
However, OOP is not only a syntax but rather a set of principles to follow.
Current approach qualifies at this level, but it won't be sufficient when you are back to your Tiny JS World in OOP Exercise task.

Reply:
So should I change the code now to implement OOP principles, or it is better to do it later?

Reply:
@kt-std you did well. You will have an opportunity to further improve your world code after a while.